vagrant-spk

```diff
@@ -356,24 +355,52 @@ def ssh(args):
     call_vagrant_command(sandstorm_dir, "ssh")
 
 def main():
-    operations = {
-            'setupvm': setup_vm,
-            'up': bring_up_vm,
-            'init': init,
-            'dev': dev,
-            'pack': pack,
-            'publish': publish,
-            'halt': halt,
-            'destroy': destroy,
-            'global-status': global_status,
-            'ssh': ssh,
-    }
-    parser = argparse.ArgumentParser(prog=sys.argv[0])
-    parser.add_argument("command", choices=sorted(operations.keys()))
+    operations = (
+        ('setupvm', setup_vm,
+            "Set up a fresh app environment in \n"
+            "  --work-directory (using the provided `stack` \n"
+            "  command arg).  May also init credentials in \n"
```

Comment:
Adding keys to the keyring actually happens over in `init` - can you move that comment over there, and make it unconditional?